fastly/vcl.go

```diff
@@ -134,8 +134,9 @@ type CreateVCLInput struct {
 	Service string
 	Version int
 
-	Name    string `form:"name,omitempty"`
-	Content string `form:"content,omitempty"`
+	Name    string       `form:"name,omitempty"`
+	Content string       `form:"content,omitempty"`
+	Main    *Compatibool `form:"main,omitempty"`
```

Comment:
I think this can just be `bool`

Reply:
ok force pushed as 8ecc9d24bfc7dc55d8decadb432a7e467363074c